### ISSUE TITLE
Failed: Error while waiting for Protractor to sync with the page: "window.angular is undefined.

### ISSUE THREAD

**Issue by brobee** [angular/protractor#2854]
Hi,

I've just started to use Protractor. I created several page objects and everything is working fine till now. I log in to a page and I'm being redirected to the myaccount page.

On that page, I got the following error: Error while waiting for Protractor to sync with the page: "window.angular is undefined.

My code is here:

var myaccount = function(){

//some function but everything is commented out
};
module.exports = new myaccount();
Here is my login test as well:

    this.loginSuccess = function(){
    userName.sendKeys(params.registration.email);
    password.sendKeys(params.registration.password);
    submitButton.click();
};
After the click, myaccount page appears but the protractor throws the mentioned error.

It seems, the myaccount page is not angularjs, there is no ng-app or other angular related stuff. I've found several topic in this question, but in this case, the rootElement config does not help for me.

How can I handle this situation?

Could somebody help me with this? Thank you in advance!

**Comment by juliemr** [comment]
Please direct general support questions like this one to an appropriate support channel, see https://github.com/angular/protractor/blob/master/CONTRIBUTING.md#questions

Thank you!

Quick answer - you need to use `browser.driver` or `ignoreSynchronization` to work on a non-angular page.

**Comment by juliemr** [comment]
See also https://github.com/angular/protractor/blob/master/docs/faq.md#how-do-i-deal-with-my-log-in-page

**Comment by brobee** [comment]
Hi Julie, thanks. I've tried it. The main problem is that, within an it() testcase, I start a non angular page but after one click, I redirected to an angular page and I got this error: Failed: unknown error: angular is not defined. Can I handle both cases in one testcase? (Nothing special, login, myaccount, search forms where the login and search is angular and my account is non angular) Strange situation!
